tensorrt_llm/_torch/compilation/utils.py

```diff
@@ -43,6 +44,16 @@ def get_enable_piecewise_cuda_graph_capture_flag() -> bool:
     return _enable_piecewise_cuda_graph_capture
 
 
+@contextlib.contextmanager
+def piecewise_cuda_graph_capture(enable: bool):
```

Comment:
Just a minor issue, maybe just personal taste: Compared to
```
with piecewise_cuda_graph_capture(True): ...
```
Will the following be better? WDYT
```
with capture_piecewise_cuda_graph(True): ...
```